themes/harbor/layouts/partials/footer.html

```diff
@@ -10,7 +10,7 @@
           <img src="{{ printf "img/logos/%s" $image | absURL }}">
         </div>
         <div class="footer-copyright">
-          <p>&copy; 2018 The Harbor Authors</p>
+          <p>&copy; 2018 Project Harbor Authors</p>
```

Comment:
Confirmed the CNCF that we should use "The Harbor Authors" to match projects like K8s. Any objections?

Reply:
I took the line from @dankohn over at https://github.com/goharbor/goharbor.github.io/issues/2. I can remove this :)

Reply:
We'd recommend:
```html
<p>&copy; Project Harbor Authors</p>
```

Reply:
Sounds good, thanks @dankohn.